### PR TITLE
Added compass.md 

### PR DESCRIPTION
**Pull request by coder135792468**
#60  added compass.md file

https://github.com/coder135792468/pslab-documentation/blob/master/compass.md

### CODE REVIEW

**Comment by cweitat** [comment]
please take care of the styling and typo errors

**Comment by pranavkarthik10** [comment]
LGTM but I think this should be in the tutorials folder.

**Comment by coder135792468** [comment]
@cweitat plz take a look again

File: compass.md [hunk 1]
```diff
@@ -1,26 +1,21 @@
 # pslab-Compass
```
Comment (cweitat) [inline compass.md]: PSLab Compass

File: compass.md [hunk 2]
```diff
@@ -1,26 +1,21 @@
...
 
-a) I proposed an experiment to find how mobile compasses work without actually using magnets within the phone.
+  Introduction: I propose an experiment on psLab using current .
 
-b) Observations :
-1) For the given experiment we need a coiled wire connected to a battery. The battery should be turn  switch on so that current starts flowing through the circuit.
+  Requirements: PsLab installed with a compass sensor.
+  
+  Observations : 
+     1. When I Pass a wire through compass then I found no deflection in the compass.
```
Comment (cweitat) [inline compass.md]: When I pass

File: compass.md [hunk 3]
```diff
@@ -1,26 +1,21 @@
 # pslab-Compass
 PSLab Documentation and Experiments https://docs.pslab.io
 
-a) I proposed an experiment to find how mobile compasses work without actually using magnets within the phone.
+  Introduction: I propose an experiment on psLab using current .
 
-b) Observations :
-1) For the given experiment we need a coiled wire connected to a battery. The battery should be turn  switch on so that current starts flowing through the circuit.
+  Requirements: PsLab installed with a compass sensor.
```
Comment (cweitat) [inline compass.md]: PSLab. Change this for other parts of document

File: compass.md [hunk 4]
```diff
@@ -1,26 +1,21 @@
...
+  
+  Observations : 
+     1. When I Pass a wire through compass then I found no deflection in the compass.
+![compass1](Screenshot_2020-01-05-20-34-03-48.png)
 
-2) Now on placing the PSLab compass near the coiled wire we notice no deflection in the compass. But on taking a normal magnetic compass we do see deflection.
-3) As we shift the normal magnetic compass around the coiled wire the compass deflects in a different manner. But this shows no effect on the PSLab compass.
-4) Thus through this experiment it shows that flowing charges (current) do produce a magnetic field which do interact with the magnet inside a normal but that show no effect on the PSLab compass.
+     2. When I pass current through the wire then I notice a deflection in the compass.
```
Comment (cweitat) [inline compass.md]: why highlighted text. If unsure what I'm talking about, have a look here https://github.com/coder135792468/pslab-documentation/blob/master/compass.md

File: compass.md [hunk 5]
```diff
@@ -1,26 +1,21 @@
...
 
-c) Conclusion :
+![compass2](Screenshot_2020-01-06-10-56-47-15.png)
```
Comment (cweitat) [inline compass.md]: reduce sizing

File: compass.md [hunk 6]
```diff
@@ -1,26 +1,21 @@
...
+  Results : PsLab compass shows deflection when I pass current through wire. 
 
-1)Without Current:
-![compass1](https://user-images.githubusercontent.com/58810632/71679145-e3eff480-2dac-11ea-8dd2-f9e764e93d11.jpeg)
-2)With Current:
-![compass2](https://user-images.githubusercontent.com/58810632/71679169-f1a57a00-2dac-11ea-9959-007f50f315a5.jpeg)
-![compass3](https://user-images.githubusercontent.com/58810632/71679202-0550e080-2dad-11ea-942a-fc88f415ada8.jpeg)
 
-e) As you can see there is no deflection caused by the flow of charges in the circuit, i.e, the induced magnetic field does interact with the PSLab compass.
+ Normal pic:
```
Comment (cweitat) [inline compass.md]: Don't use short words "pic" . Use "picture"

**Comment by cweitat** [comment]
and move to tutorials folder

**Comment by NishithP2004** [comment]
There are many Grammatical errors as well.

**Comment by NishithP2004** [comment]
Please don't deleted the present data and create a new section for your new exp.

**Comment by NishithP2004** [comment]
Please don't deleted the present data and create a new section for your new exp.

**Comment by coder135792468** [comment]
What I have to do please anyone explain what I have to do after correction of the Grammer errors? 
Please provide all the information that I have to change

**Comment by NishithP2004** [comment]
You have to copy your experiment to a doc and then revert all the changes
you've done, then create a new section in the compass.md file by adding
your experiment as ## Experiment 3 after ## Experiment 2.

**Comment by NishithP2004** [comment]
You have to add your emperiment sequential as how it's present in the file.

**Pull request closed without merging**